js/bbs-api/index.html

```diff
@@ -6,6 +6,7 @@
   </head>
   <body>
     <table>
+      <div id="test"></div>
```

Comment:
[must]
js側のコードを読むと、ここにメッセージを埋め込む形でしょうか

`test`だとどういったエレメントかわからないので命名を工夫してみましょう

例）
```suggestion
      <div id="message-area"></div>
```